src/main/java/blackjack/domain/game/result/PlayerLose.java

```diff
@@ -0,0 +1,40 @@
+package blackjack.domain.game.result;
+
+import blackjack.domain.betting.Money;
+import blackjack.domain.participant.Dealer;
+import blackjack.domain.participant.Player;
+
+public class PlayerLose implements GameResult {
+
+    private static final PlayerLose instance = new PlayerLose();
+
+    private PlayerLose() {
+    }
+
+    //TODO 라인 길이 수정
+    @Override
+    public boolean isSatisfy(Dealer dealer, Player player) {
+        if (player.isBusted()) {
+            return true;
+        }
+        if (player.isBlackjack()) {
+            return false;
+        }
+        if (dealer.isBlackjack()) {
+            return true;
+        }
+        if (dealer.isBusted()) {
+            return false;
+        }
+        return player.calculateScore() < dealer.calculateScore();
+    }
+
+    @Override
+    public Money calculatePrize(Money bet) {
+        return bet.multiply(-1);
```

Comment:
```suggestion
        return bet.negate();
```
부르는 쪽에서의 인지적 부담을 최소화합시다!

Reply:
와, 이렇게 적어줄 수도 있었네요! 감사합니다 👍